symfony/plugins/orangehrmCorePlugin/Service/DateTimeHelperService.php

```diff
@@ -26,7 +26,7 @@
 
 class DateTimeHelperService
 {
-    public const TIMEZONE_UTC = 'UTC';
+    public const TIMEZONE_UTC = '+0000';
```

Comment:
@samanthajayasinghe need a help here. Used "UTC" according to https://www.php.net/manual/en/timezones.others.php, but it's not working on testing servers. Couldn't found root cause. Do you have any idea.

Requirement: Get calculated UTC time from the server
Expected: Correct UTC time, not based on server timezone
Actual: Getting server time, without converting to UTC. (Not reproducible in dev environment)

```php
$currentDateTime = $this->getDateTimeHelper()->getNow(new DateTimeZone('UTC'));
```